depends/packages/zeromq.mk

```diff
@@ -3,7 +3,7 @@ $(package)_version=4.3.1
 $(package)_download_path=https://github.com/zeromq/libzmq/releases/download/v$($(package)_version)/
 $(package)_file_name=$(package)-$($(package)_version).tar.gz
 $(package)_sha256_hash=bcbabe1e2c7d0eec4ed612e10b94b112dd5f06fcefa994a0c79a45d835cd21eb
-$(package)_patches=remove_libstd_link.patch
+$(package)_patches=remove_libstd_link.patch fix_dash_compat.patch
```

Comment:
Apply patch in `$(package)_preprocess_cmds`?

Reply:
Thanks. Adjusted the patch as well.